src/main/groovy/org/mockito/release/internal/gradle/VersioningPlugin.java

```diff
@@ -30,6 +31,9 @@
  * </ul>
  *
  * Also, the plugin configures all projects' version property to the value specified in "version.properties"
+ *
+ * BEWARE! If version.properties doesn't exists, this plugin will create it automatically and set
```

Comment:
Nice javadoc! The Javadoc descriptions of our plugins will be a part of our official documentation. It's great to describe exactly what happens in the plugin.